src/components/Accordion/Accordion.tsx

```diff
@@ -3,8 +3,9 @@ import React, { useId } from 'react';
 import type { ClickHandler } from './Context';
 import { AccordionIconVariant, AccordionContext } from './Context';
 import classes from './Accordion.module.css';
+import type { AccordionHeaderProps } from './AccordionHeader';
 
-export interface AccordionProps {
+export interface AccordionProps extends AccordionHeaderProps {
```

Comment:
This should be removed as well, since the interface for the Accordion was not changed after all

Reply:
Oops, forgot that one 😅
Done now 😊